core/src/trezor/utils.py

```diff
@@ -169,6 +169,42 @@ def extend(self, __buf: bytes) -> None:
             ...
 
 
+if __debug__:
```

Comment:
Make this `if False` please. We don't want dead code lying around, even in debug builds (which are larger just by themselves)

Reply:
Rebased over master and force-pushed.